### ISSUE TITLE
XSS on Gila CMS Installation

### ISSUE THREAD

**Issue by sie504** [GilaCMS/gila#41]
**XSS on Gila CMS Installation**

**Gila CMS version 1.11.3** 

1: Admin Username

![](https://image-1258195556.cos.ap-shanghai.myqcloud.com/new/Snip20190924_223.png)

`<input name="adm_user" placeholder="Your Name" required="">`

adm_user

/install/install.sql.php

```
$_user=$_POST['adm_user'];
$_email=$_POST['adm_email'];
$_pass=password_hash($_POST['adm_pass'], PASSWORD_BCRYPT);

$link->query("INSERT INTO userrole(id,userrole) VALUES(1,'Admin');");
$link->query("INSERT INTO user(id,username,email,pass,active,reset_code) VALUES(1,'$_user','$_email','$_pass',1,'');");
```

2:Login in admin pane

XSS

![](https://image-1258195556.cos.ap-shanghai.myqcloud.com/new/Snip20190924_224.png)

Visit the website

![](https://image-1258195556.cos.ap-shanghai.myqcloud.com/new/Snip20190924_225.png)

3:Reference

https://www.owasp.org/index.php/Cross-site_Scripting_(XSS)

**Comment by vzuburlis** [comment]
Thank you @sie504 for the report. Fields should filter the input values

**Comment by vzuburlis** [comment]
This issue is fixed on v1.11.4. Closing
